.devcontainer/Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM mcr.microsoft.com/devcontainers/base:alpine-3.18
 # Setzen der Umgebungsvariablen
 ENV FIRELY_TERMINAL_VERSION=3.1.0
 ENV JAVA_VALIDATOR_VERSION=6.0.11
-ENV SUSHI_VERSION=3.5.0
-
+ENV SUSHI_VERSION=3.7.0
```

Comment:
@patrick-werner generell sushi hochziehen in allen repos?

Reply:
würde ich so vorschlagen, ja.